Provision/base.sql

```diff
@@ -65,7 +65,8 @@ CREATE TABLE IF NOT EXISTS summoners
 CREATE TABLE IF NOT EXISTS matches
 (
     id INT NOT NULL AUTO_INCREMENT,
-    role ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
+    match_id INT NOT NULL,
+    lane ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
```

Comment:
We could always parse it into what we consider viable, but if it is unreliable anyway we can skip it for now.